tests/unit/test_compositions.py

```diff
@@ -62,6 +66,24 @@ def it_knows_tissue_areas_mask_slide_filters_composition(
     assert type(_enough_tissue_mask_filters_) == Compose
 
 
+def it_knows_tissue_areas_mask_compose_filters_composition(
+    RgbToGrayscale_,
+    OtsuThreshold_,
+    BinaryDilation_,
+    RemoveSmallHoles_,
+    RemoveSmallObjects_,
+):
+    _enough_tissue_mask_filters_ = FiltersComposition(
+        Compose,
+        RgbToGrayscale_,
+        OtsuThreshold_,
+        BinaryDilation_,
+        RemoveSmallHoles_,
+        RemoveSmallObjects_,
+    ).tissue_mask_filters
+    assert type(_enough_tissue_mask_filters_) == Compose
```

Comment:
IMHO this test is not meaningful. If you take a look at lines 52-58 you will see the assertions for the mocks. Could you please integrate those also in this test? Thanks